### DISCUSSION TITLE
[Request] Street Fighter VI - Unlock Framerate During Matches

### DISCUSSION THREAD

**Discussion by RaygeSol** [praydog/REFramework#765] [General Questions]
The game is capable of 120fps seen from the menu, character select, and world tour mode. But framerates are then locked to 60FPS during matches. I was wondering if anyone could look into this. I don't have any skills with this sort of thing. I tried using different .dlls and also tried to see if I could the risetweaks.lua (has an option to unlock framerates) to work in SF6, but have had no success.

**Comment by praydog** [comment]
Tried this briefly. All it does is make the game run at a different speed as a consequence.

**Reply by RaygeSol** [reply]
Thanks for the update! Was hoping that wouldn't be the case.
